website/static/js/registrationUtils.js

```diff
@@ -339,7 +344,7 @@ var Draft = function(params, metaSchema) {
             $.each(schema.pages, function(i, page) {
                 $.each(page.questions, function(qid, question) {
                     var q = self.schemaData[qid];
-                    if (q && (q.value || '').trim()) {
+                    if (q && (q.value || '').trim() !== '') {
```

Comment:
Will this completion counter break if encounters fields whose value is not a string? (eg a checkbox)

Can you clarify why the extra comparison vs empty string was added here? (vs checking whether value was falsy)